### PR TITLE
chore: Remove webgl imports 2

### PR DESCRIPTION
**Pull request by ibgreen**
For #

#### Background
- Remove all webgl module imports
#### Change List
- Use new device methods in luma.gl beta.2

### CODE REVIEW

File: modules/extensions/package.json [hunk 1]
```diff
@@ -32,15 +32,15 @@
...
-    "@luma.gl/constants": "9.0.0-beta.1",
-    "@luma.gl/core": "9.0.0-beta.1",
-    "@luma.gl/engine": "9.0.0-beta.1",
-    "@luma.gl/shadertools": "9.0.0-beta.1",
-    "@luma.gl/webgl": "9.0.0-beta.1",
+    "@luma.gl/constants": "9.0.0-beta.3",
+    "@luma.gl/core": "9.0.0-beta.3",
+    "@luma.gl/engine": "9.0.0-beta.3",
+    "@luma.gl/shadertools": "9.0.0-beta.3",
+    "@luma.gl/webgl": "9.0.0-beta.3",
```
Comment (Pessimistress) [inline modules/extensions/package.json:43]: This should no longer be needed?

File: modules/google-maps/package.json [hunk 2]
```diff
@@ -38,10 +38,10 @@
...
   "peerDependencies": {
     "@deck.gl/core": "^9.0.0-alpha",
-    "@luma.gl/constants": "9.0.0-beta.1",
-    "@luma.gl/core": "9.0.0-beta.1",
-    "@luma.gl/engine": "9.0.0-beta.1",
-    "@luma.gl/webgl": "9.0.0-beta.1",
+    "@luma.gl/constants": "9.0.0-beta.3",
+    "@luma.gl/core": "9.0.0-beta.3",
+    "@luma.gl/engine": "9.0.0-beta.3",
+    "@luma.gl/webgl": "9.0.0-beta.3",
```
Comment (Pessimistress) [inline modules/google-maps/package.json:44]: This should no longer be needed?

File: modules/extensions/package.json (hunk 1 above)
Comment (ibgreen) [reply modules/extensions/package.json:43]: Extensions is still using WEBGLRenderBuffer. I have asked @felixpalmer if he can update CollisionFilterExtension to use a depth texture instead.

File: modules/google-maps/package.json (hunk 2 above)
Comment (ibgreen) [reply modules/google-maps/package.json]: Should be removed now

File: modules/core/src/passes/pick-layers-pass.ts [hunk 3]
```diff
@@ -77,8 +75,7 @@ export default class PickLayersPass extends LayersPass {
     // Note that the callback here is called synchronously.
     // Set blend mode for picking
     // always overwrite existing pixel with [r,g,b,layerIndex]
-    const renderStatus = withGLParameters(
-      this.device,
+    const renderStatus = this.device.withParametersWebGL(
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/core/src/passes/pick-layers-pass.ts:78]: Responded to the comment in #8475, yes all these calls on the Device that end in the `WebGL` suffix are not available on WebGPU and deck.gl needs to migrate away from them. 

I expect we will remove them in 9.1.

File: modules/extensions/package.json (hunk 1 above)
Comment (felixpalmer) [reply modules/extensions/package.json:50]: https://github.com/visgl/deck.gl/pull/8477

**Review by felixpalmer — APPROVED** [review]

**Pull request merged**